src/main/java/com/dawsonsystems/session/MongoManager.java

```diff
@@ -593,9 +485,9 @@ private void initDbConnection(String path) throws LifecycleException {
       }
       db.setWriteConcern(WriteConcern.ACKNOWLEDGED);
       getCollection().createIndex(new BasicDBObject("lastmodified", 1));
-      log.info("Connected to Mongo " + host + "/" + database + " for session storage, slaveOk=" + slaveOk + ", " + (getMaxInactiveInterval() * 1000) + " session live time");
+      log.info("Connected to Mongo " + host + "/" + database + " for session storage, slaveOk=" + slaveOk + ", " + (getSessionMaxAliveTime() * 1000) + " session live time");
     } catch (RuntimeException | IOException e) {
-      e.printStackTrace();
+      log.severe(String.valueOf(e.getStackTrace()));
```

Comment:
Changed per @ispringer 's comments on original PR https://github.com/evergage/Mongo-Tomcat-Sessions/pull/7. 

but is it possible the exception was thrown due to writing logs? In that case, the error could get lost. BTW, the e.printStackTrace() code was from the original code we forked from.

Reply:
Use https://docs.oracle.com/javase/8/docs/api/java/util/logging/Logger.html#log-java.util.logging.Level-java.lang.String-java.lang.Throwable- instead. Let the logger handle stringifying the exception.